contracts/BatchExchange.sol

```diff
@@ -241,16 +246,16 @@ contract BatchExchange is EpochTokenLocker {
         uint32[] memory validUntils,
         uint128[] memory buyAmounts,
         uint128[] memory sellAmounts
-    ) public returns (uint256[] memory orderIds) {
+    ) public returns (uint16[] memory) {
         cancelOrders(cancellations);
         return placeValidFromOrders(buyTokens, sellTokens, validFroms, validUntils, buyAmounts, sellAmounts);
     }
 
     /** @dev a solver facing function called for auction settlement
       * @param batchId index of auction solution is referring to
       * @param owners array of addresses corresponding to touched orders
-      * @param orderIds array of order ids used in parallel with owners to identify touched order
-      * @param buyVolumes executed buy amounts for each order identified by index of owner-orderId arrays
+      * @param orderIndices array of order ids used in parallel with owners to identify touched order
```

Comment:
```suggestion
      * @param orderIds an array of order indices used in parallel with owners to identify touched order
```

Reply:
Sorry, but for me it makes no sense to call it `Ids` in the name and then `indices` the comments.

But good catch, some comments were not yet updated

Reply:
variable name `somethingIds` followed by a description of what the variable is (i.e. indices of an array) has no conceptual conflict. In fact, I would argue that it makes the description more meaningful and rich. In the same respect that one would not define a term with the term itself.